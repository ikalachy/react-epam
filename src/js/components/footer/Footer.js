import React, { Component } from "react";

const Footer = (props) =>
    <footer>
        <p className="text-center"> copyright (c) 2020</p>
        {props.children}
    </footer>

export default Footer