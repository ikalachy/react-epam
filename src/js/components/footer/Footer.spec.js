import React from 'react'
import renderer from 'react-test-renderer'

import Footer from './Footer'

it('render Footer component without children', () => {
    const actualSnapshot = renderer
        .create(<Footer />)
        .toJSON()

    expect(actualSnapshot).toMatchSnapshot()
})

it('render Footer component with children', () => {
    const actualSnapshot = renderer
        .create(
            <Footer>
                <h1>Hello, Footer</h1>
            </Footer>)
        .toJSON()

    expect(actualSnapshot).toMatchSnapshot()
})

