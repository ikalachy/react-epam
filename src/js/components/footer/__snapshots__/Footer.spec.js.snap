// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Footer component with children 1`] = `
<footer>
  <p
    className="text-center"
  >
     copyright (c) 2020
  </p>
  <h1>
    Hello, Footer
  </h1>
</footer>
`;

exports[`render Footer component without children 1`] = `
<footer>
  <p
    className="text-center"
  >
     copyright (c) 2020
  </p>
</footer>
`;
